### PR TITLE
calibrate: shorten the minimum calibrate duration

### PR DESCRIPTION
**Pull request by CabinfeverB**
### What problem does this PR solve?

Issue Number: ref #43212

Problem Summary:

### What is changed and how it works?
shorten the minimum calibrate duration

should be merged after #44965 because of fixing test bug.

### Check List

Tests 

- [x] Unit test
- [ ] Integration test
- [ ] Manual test (add detailed scripts or steps below)
- [ ] No code

Side effects

- [ ] Performance regression: Consumes more CPU
- [ ] Performance regression: Consumes more Memory
- [ ] Breaking backward compatibility

Documentation

- [ ] Affects user behaviors
- [ ] Contains syntax changes
- [ ] Contains variable changes
- [ ] Contains experimental features
- [ ] Changes MySQL compatibility

### Release note

Please refer to [Release Notes Language Style Guide](https://pingcap.github.io/tidb-dev-guide/contribute-to-tidb/release-notes-style-guide.html) to write a quality release note.

```release-note
None
```

### CODE REVIEW

**Comment by CabinfeverB** [comment]
cc @glorv @nolouch

File: executor/internal/calibrateresource/calibrate_resource.go [hunk 1]
```diff
@@ -105,14 +105,12 @@ const (
...
 	lowUsageThreshold = 0.1
-	// calibration is performed only when the available time point exceeds the percentOfPass
-	percentOfPass = 0.9
 	// For quotas computed at each point in time, the maximum and minimum portions are discarded, and discardRate is the percentage discarded
 	discardRate = 0.1
 
 	// duration Indicates the supported calibration duration
 	maxDuration = time.Hour * 24
-	minDuration = time.Minute * 10
+	minDuration = time.Minute * 1
```
Comment (glorv) [inline executor/internal/calibrateresource/calibrate_resource.go:113]:
```suggestion
	minDuration = time.Minute
```

**Review by glorv — APPROVED** [review]
LGTM

**Review by nolouch — APPROVED** [review]

**Pull request merged**